docs/nginx.md

```diff
@@ -33,6 +33,24 @@ While this model gives you the ability to use whatever authentication backend yo
 
 Furthermore, introducing an extra http layer in your communication pipeline will make it more complex to deploy, maintain, and debug, and will possibly create issues.
 
+For instance, Amazon's Elastic Load Balancer (ELB) already sets the following client header:
```

Comment:
This only works when ELB mode is HTTP or HTTPS. May want to add a link here to the ELB docs.